### ISSUE TITLE
Test building on Windows and create instructions for Windows

### ISSUE THREAD

**Issue by rdb** [panda3d/panda3d-docs#38]
I had an exchange with a user on Discord who struggled to build the manual properly on Windows.  We need to make sure that the instructions are correct for Windows and test that there aren't problems with the build on Windows.

**Comment by serkkz** [comment]
I suggest dividing into 3 specific steps.

1. Installation, Sphinx. You can give a link to the office website. Or skip the step if one is installed.

2. Getting resources, text files, images, source code, and more.

3. Build in the desired format, such as HTML.

```
Building The Documentation
pip install -r requirements.txt
make html
```
This is very, very not informative. The transparency of the process is lost.

**Comment by rdb** [comment]
Yes, we need to work on improving the documentation.  The current was just a stand-in.

The issue you were running into (with the path to the .css being wrong) was due to a bug in the [sphinx-variations](https://github.com/treamology/sphinx-variations) extension on Windows.  I will ensure that this bug is fixed upstream shortly.

**Comment by rdb** [comment]
I have fixed the sphinx-variations bug in 47770e83bfb0c10bd6d2f34ffcfb493a5077a724 and published that as 1.0.4.  You may want to update your sphinx-variations version; the pip command should do that automatically if you update to the latest Git version, or you can do it explicitly with `pip install -U sphinx-variations`.
